VERSION.md

```diff
@@ -1 +1 @@
-1.3.4
+1.4.2
```

Comment:
Not necessary to edit this file by now. It's part of an automation that is not working (wip).

Reply:
got, revert the change